src/selectors/ariaAttributes.js

```diff
@@ -57,7 +57,7 @@ function calculateAriaAttributes(rowsCount, useGroupHeader, useFooter) {
   };
 }
 
-export default shallowEqualSelector([
+export default createSelector([
```

Comment:
can you explain why `createSelector` was used instead of `shallowEqualSelector` ?